addOns/retire/src/test/java/org/zaproxy/addon/retire/RetireScanRuleUnitTest.java

```diff
@@ -268,7 +277,18 @@ void shouldHaveExpectedExampleAlert() {
         List<Alert> alerts = rule.getExampleAlerts();
         // Then
         assertThat(alerts.size(), is(equalTo(1)));
-        assertThat(alerts.get(0).getRisk(), is(equalTo(Alert.RISK_MEDIUM)));
+        Alert example = alerts.get(0);
+        assertThat(example.getRisk(), is(equalTo(Alert.RISK_MEDIUM)));
+        assertThat(
+                example.getDescription(),
+                is(equalTo("The identified library appears to be vulnerable.")));
+        assertThat(example.getEvidence(), is(equalTo("13.3.7")));
+        assertThat(
+                example.getOtherInfo(),
+                is(
+                        equalTo(
+                                Constant.messages.getString(
```

Comment:
Since we are hardcoding the text in other assert (i.e. `shouldRaiseAlertOnHashOfVulnerableContent´) why not here as well?

Reply:
There isn't a good reason, this is just the mechanism that was in my head at the time. I'll "try" to be more consistent in the future.